src/transactions/transaction_db.rs

```diff
@@ -931,6 +931,24 @@ impl<T: ThreadMode> TransactionDB<T> {
     pub fn snapshot(&self) -> SnapshotWithThreadMode<Self> {
         SnapshotWithThreadMode::<Self>::new(self)
     }
+
+    fn drop_column_family<C>(
+        &self,
+        cf_inner: *mut ffi::rocksdb_column_family_handle_t,
+        cf: C,
+    ) -> Result<(), Error> {
+        unsafe {
+            // first mark the column family as dropped
+            ffi_try!(ffi::rocksdb_drop_column_family(
+                self.inner as *mut ffi::rocksdb_t,
+                cf_inner
+            ));
+        }
+        // then finally reclaim any resources (mem, files) by destroying the only single column
+        // family handle by drop()-ing it
+        drop(cf);
```

Comment:
Is this only for readability? The handle will be dropped anyway. Personally I would leave the comment and remove the `drop` function call, but it is up to you.

Reply:
done